src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs

```diff
@@ -115,13 +115,13 @@ protected ReindexJobRecord()
         public IReadOnlyCollection<string> TargetResourceTypes { get; private set; } = new List<string>();
 
         [JsonIgnore]
-        public int PercentComplete
+        public decimal PercentComplete
         {
             get
             {
                 if (Count > 0 && Progress > 0)
                 {
-                    return (int)((double)Progress / Count * 100);
+                    return (decimal)Math.Round((double)Progress / Count * 100, 1);
```

Comment:
>decimal [](http://example.com/codeflow?start=28&length=7)

super nit:
why cast it to `decimal` if it's already `double`?

Especially if we display only one digit ;)